.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Documentation request
+about: Ask for documentation of a feature
+title: "Docs: ..."
+labels: type/docs
+assignees: ""
+---
+
+#### Is your documentation request related to a feature? Which one?
+
+A clear and concise description of the feature you were trying to use, and what you were trying to achieve.
+
+#### Describe the solution you'd like
+
+A clear and concise description of the documentation you are missing.
```

Comment:
@chri2547, please review this. :)

Reply:
Sorry @osg-grafana , I'm late to the party! I think this looks fine. Glad to have this issue template. Thanks, all!